torch/jit/__init__.py

```diff
@@ -1997,6 +1997,7 @@ def _unwrap_optional(x):
     return x
 
 
+unimplemented_math_ops = ["fsum", "isclose", "trunc"]
```

Comment:
Is it intended to make `unimplemented_math_ops` available in `torch.jit` namespace? This is user facing, why do we want this?

Reply:
The goal was to not duplicate the list across `__init__` and `test_jit`: https://github.com/pytorch/pytorch/pull/21511#discussion_r298387966

Is there another namespace I can put this in?

Reply:
I'm not sure if there's another namespace hmmm, I commented below to check if this behavior is expected or not.